RSyntaxTextArea/src/test/java/org/fife/ui/rsyntaxtextarea/folding/PythonFoldParserTest.java

```diff
@@ -23,34 +23,37 @@ class PythonFoldParserTest {
 	@Test
 	void testGetFold_happyPath() {
 
-		String code = "#!/usr/bin/env python\n" +
-			"\n" +
-			"import getopt, sys, urllib, time\n" +
-			"def main():\n" +
-			"\n" +
-			"    status = 0\n" +
-			"\n" +
-			"# input arguments\n" +
-			"\n" +
-			"    try:\n" +
-			"    opts, args = getopt.getopt(sys.argv[1:],\"h:iq\",\n" +
-			"                   [\"help\",\"invid=\",\"quarter=\"])\n" +
-			"    except getopt.GetoptError:\n" +
-			"    usage()\n" +
-			"    tree = False\n" +
-			"    for o, a in opts:\n" +
-			"    if o in (\"-h\", \"--help\"):\n" +
-			"        usage()\n" +
-			"\n" +
-			"    kepid, invid, kepmag, mode, start, stop, release = GetMetaData(invid,quarter)\n" +
-			"\n" +
-			"# convert Gregorian date to Julian date\n" +
-			"\n" +
-			"def Greg2JD(year, month, day):\n" +
-			"\n" +
-			"    if (month < 3):\n" +
-			"        y = float(year) - 1.0\n" +
-			"        m = float(month) + 12.0\n";
```

Comment:
You'll have to put this back, RSTA is built with Java 8 :(

Reply:
Oops.  Reverted back and added line continuation support.  Will commit with other requested changes.